src/igvfd/tests/test_schema_in_vitro_system.py

```diff
@@ -21,3 +21,40 @@ def test_time_post_factors_dependency(in_vitro_differentiated_cell, testapp):
         in_vitro_differentiated_cell['@id'],
         {'time_post_factors_introduction': 3, 'time_post_factors_introduction_units': 'day'}, expect_errors=True)
     assert res.status_code == 200
+
+
+def test_sorted_fraction(testapp, lab, award, source, human_donor, sample_term_K562):
+    item_1 = {
+        'classification': 'differentiated cell',
+        'award': award['@id'],
+        'lab': lab['@id'],
+        'source': source['@id'],
+        'taxa': 'Homo sapiens',
+        'donors': [human_donor['@id']],
+        'biosample_term': sample_term_K562['@id']
+    }
+    source_cell = testapp.post_json('/in_vitro_system', item_1, status=201).json['@graph'][0]
```

Comment:
I am not sure what this post is testing?

Reply:
used fixtures instead of posting